compiler/test/fail_compilation/test20245.d

```diff
@@ -9,7 +9,7 @@ fail_compilation/test20245.d(27): Error: cannot take address of `scope` variable
 fail_compilation/test20245.d(33): Error: reference to local variable `x` assigned to non-scope parameter `ptr` calling `escape`
 fail_compilation/test20245.d(34): Error: copying `&x` into allocated memory escapes a reference to parameter `x`
 fail_compilation/test20245.d(50): Error: reference to local variable `price` assigned to non-scope `this.minPrice`
-fail_compilation/test20245.d(69): Error: reference to local variable `this` calling non-scope member function `this.this()`
+fail_compilation/test20245.d(69): Error: reference to local variable `this.content[]` calling non-scope member function `Exception.this()`
```

Comment:
Here I used `arg` because I don't know how to get the `content` variable from.

Reply:
Yes, the escapeBy results discard the information which struct member is actually being escaped, which also gives problems with the hasPointers check in some cases. I want to improve that at some point.